index.js

```diff
@@ -1,20 +1,15 @@
 // const passElement = document.getElementById('pass').value;
 const decodeButton = document.querySelector('.btn').onclick = function startDecode() {
 
-        // while (i < 10000) {
-        //     if( i === passElement.value)
-        //     alert( i )
-        //     i++
-        //     break
-        // }
-    let i = 0
-if (i < passElement.value) {i++}
-if (i = passElement.value) alert( i )
+        while (i < 10000) {
+        if( i == passElement.value) {
+            break;
+        } 
+        i = i + 1}
```

Comment:
`}` should be on new line, att first glance I didn't understand how it works, a bit confused.
Also you should keep tabulation 

`
```javascript
        while (i < 10000) {
           if( i == passElement.value) {
               break;
           } 
           i = i + 1
        }
```